api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,14 +117,29 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+	<changeSet id="20180501-TRUNK-5338" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="conditions"/>
+		</preConditions>
+		<comment>
+			Modify conditions table to match condition object in the core platform if the conditions table already exists in the database
+		</comment>
+		<renameColumn columnDataType="int" newColumnName="previous_version" oldColumnName="previous_condition_id" tableName="conditions"/>
+		<renameColumn columnDataType="varchar(255)" newColumnName="clinical_status" oldColumnName="status" tableName="conditions"/>
+		<renameColumn columnDataType="int" newColumnName="condition_coded" oldColumnName="concept_id" tableName="conditions"/>
+
+		<addColumn tableName="conditions">
+			<column name="condition_coded_name" type="int"/>
+		</addColumn>
+	</changeSet>
 	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
```

Comment:
It's good to change the id when you change the changeset contents so that this doesn't break other developer's working copies or our CI servers

Reply:
Yes. Done

Reply:
I don't see the change